src/js/Controllers/BCController.js

```diff
@@ -98,6 +101,18 @@ class BCController {
             case "GetSystemTime":
                 this.listener.send(RpcFactory.GetSystemTime(rpc.id))
                 return null
+
+            case "OnAppCapabilityUpdated":
+                if (rpc.params.appCapability.appCapabilityType === 'VIDEO_STREAMING') {
+                    var vsc = rpc.params.appCapability.videoStreamingCapability;
+                    var caps = [ vsc ];
+                    if (vsc.additionalVideoStreamingCapabilities) {
+                        caps.concat(vsc.additionalVideoStreamingCapabilities);
+                        delete vsc.additionalVideoStreamingCapabilities;
```

Comment:
Do you mean to delete caps.additionalVideoStreamingCapabilities? Deleting this is modifying the original rpc message and causing the console logs to show that additionalVideoStreamingCapabilities is never received.

Reply:
Also caps seems to be losing track of the additionalVideoStreaminCapabilities param
![Screen Shot 2021-02-01 at 5 20 44 PM](https://user-images.githubusercontent.com/12038416/106525333-d4604500-64b1-11eb-8484-5b8d9985be39.png)

Reply:
I stringified the request to see if the additional video params were in the request and it was. Last printed line is caps which is also missing the additional params

Reply:
Oh yea this is probably why

> The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.

Reply:
fixed in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/7d647f7bc4fd20a4ddae2717d968b2032819d4bc